tests/e2e/Services/Account/AccountCustomClientTest.php

```diff
@@ -744,8 +743,31 @@ public function testUpdateAccountEmailSearch($data): void
 
     public function testCreatePhone(): array
     {
-        $number = '+123456789';
+        $to = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_TO');
+        $from = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_FROM');
+        $authKey = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_AUTH_KEY');
+        $senderId = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_SENDER_ID');
 
+        if ($to === '' || $from === '' || $authKey === '' || $senderId === '') {
```

Comment:
Let's check using `empty` here instead of `=== ''`, the default for `getEnv` is null